pandas-stubs/util/_validators.pyi

```diff
@@ -7,7 +7,8 @@ def validate_kwargs(fname, kwargs, compat_args) -> None: ...
 def validate_args_and_kwargs(
     fname, args, kwargs, max_fname_arg_count, compat_args
 ) -> None: ...
-def validate_bool_kwarg(value, arg_name): ...
-def validate_axis_style_args(data, args, kwargs, arg_name, method_name): ...
```

Comment:
New args for `bool_kwarg` while `validate_axis_style_args` does not exist anymore.

Reply:
I think we should delete this file.  It's private and not documented.